### PR TITLE
Allow AI to change alert level (again)

### LINKED ISSUE #13102: AI appears able to set alert levels
The AI appears to be able to set station alert levels when using a console with the command and communications program open. This may require someone else having left the program open, unsure.

I didn't actually try pressing the buttons to avoid IC disruption, but the buttons were blue and appeared clickable:

### PR DESCRIPTION
**Pull request by DreamySkrell**
https://forums.aurorastation.org/topic/17251-give-ais-full-access-to-the-command-programmes/#comment-157741

AI can change alert level
borgs cannot

fixes #13102

### CODE REVIEW

**Comment by DreamySkrell** [comment]
!review

File: code/modules/modular_computers/file_system/programs/command/comm.dm [hunk 1]
```diff
@@ -242,7 +242,7 @@
 						post_display_status(href_list["target"])
 
 		if("setalert")
-			if(is_authenticated(user) && !issilicon(usr) && ntn_cont && ntn_comm)
+			if((is_authenticated(user) && !issilicon(usr) && ntn_cont && ntn_comm) || (is_authenticated(user) && isAI(usr) && ntn_cont && ntn_comm))
```
Comment (NonQueueingMatt) [inline code/modules/modular_computers/file_system/programs/command/comm.dm:245]:
```suggestion
			if(is_authenticated(user) && (!issilicon(usr) || isAI(usr)) && ntn_cont && ntn_comm)
```

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Comment by DreamySkrell** [comment]
fixes #13102

File: html/changelogs/DreamySkrell-PR-14443.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+author: DreamySkrell
+
+delete-after: True
+
+changes:
+  - tweak: " Allow AI to change alert level (again)."
```
Comment (NonQueueingMatt) [inline html/changelogs/DreamySkrell-PR-14443.yml:6]:
```suggestion
  - tweak: "Allow AI to change alert level (again)."
```

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**